include/zephyr/drivers/display.h

```diff
@@ -279,6 +279,29 @@ static inline int display_read(const struct device *dev, const uint16_t x,
 	return api->read(dev, x, y, desc, buf);
 }
 
+/**
+ * @brief Indicates that all writes for the current frame are done
+ *
+ * Can be used to present all previous writes simultaneously, for
+ * double buffered or latched displays.
+ *
+ * @param dev Pointer to device structure
+ *
+ * @retval 0 on success else negative errno code.
+ */
+static inline int display_finalize_frame(const struct device *dev)
+{
+	struct display_driver_api *api =
+		(struct display_driver_api *)dev->api;
+
+	if (api->finalize_frame == NULL) {
+		return 0;
```

Comment:
```suggestion
		return -ENOSYS;
```

Reply:
My thought was that if a driver does not require any special operation to finalize the frame, it can simply not implement this function and it will be successful.

Reply:
+1 on the concern of trickeling LVGL functionality into the display driver api.

Reply:
Fixed